ros2_build.sh

```diff
@@ -23,7 +34,7 @@ if [ -z "${NO_TEST}" ]; then
     # run unit tests
     . ./install/setup.sh
     colcon test
```

Comment:
If I remember correctly, we should touch a `COLCON_IGNORE` under the `dep` directory before running `colcon test`, because we don't want to run the tests of the dependencies, similar to the Travis CI flow for ROS1.

Reply:
Added